pkg/util/tz.go

```diff
@@ -57,9 +57,6 @@ func GetLocalTimezone() (*time.Location, error) {
 	if time.Local.String() != "Local" {
 		return time.Local, nil
 	}
-	str, err := os.Readlink("/etc/localtime")
-	if err != nil {
-		return nil, cerror.WrapError(cerror.ErrLoadTimezone, err)
-	}
+	str := timeutil.InferSystemTZ()
 	return getTimezoneFromZonefile(str)
```

Comment:
Seems `getTimezoneFromZonefile` can be simplified after we use infer system tz.

Reply:
ok